cmd/command.go

```diff
@@ -97,24 +68,22 @@ func handleConsole(ctx context.Context, tele *telemetry.Client, cmd *cobra.Comma
 		// No console client created here
 	case "describe":
 		var err error
-		c, err = console.ClientFactory(ctx, &cfgPath, cfgMutator)
+		c, err = console.CreateClient(ctx, cfgPath, true, cfgMutator)
 		if err != nil {
 			return err
 		}
-		defer c.Client().Close()
 	case "fetch":
 		cfgMutator = filterConfigProviders(args)
 		fallthrough
 	default:
 		var err error
-		c, err = console.CreateClient(ctx, cfgPath, cfgMutator)
+		c, err = console.CreateClient(ctx, cfgPath, false, cfgMutator)
 		if err != nil {
 			return err
 		}
-		defer c.Client().Close()
 	}
 
-	if tele.Enabled() && tele.NewRandomId() {
+	if analytics.Enabled() {
```

Comment:
There's a functionality disparity here: we only do this notice/delay thing if we had created a "new" ID, so it was only shown once (unless the filesystem isn't persistent)

New code shows/delays every time

Reply:
I think it should show every time to be honest even if Id is already created.

Reply:
but it also delays for 2 seconds.

Reply:
we can live with that for now ?